### DISCUSSION TITLE
tabelle di decodifica per il multilinguismo

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#935] [Supporto servizi cooperativi] [answered]
Buongiorno,
quando verranno rese pubbliche le tabelle di decodifica anche con la descrizione in tedesco?
Ovviamente mi riferisco a quelle tabelle dove nel json bisogna riportare la descrizione, come ad esempio la 11
![image](https://github.com/italia/ansc/assets/127182190/31ec457c-d094-4385-a537-f8d2fd8585e7)

**Comment by mttfranci** [comment, marked as answer]
Buongiorno. Al momento, come in ANPR, non è prevista la pubblicazione delle tabelle in altre lingue.
Visto che vanno inviati dei codici, è sufficiente la versione italiana per determinare il valore da inviare.
Comunque valuteremo se sia possibile pianificare la pubblicazione anche delle decodifiche in altre lingue.

Saluti

Assistenza ANSC
